tetris.py

```diff
@@ -27,9 +27,12 @@ def run_gui():
     root = tkinter.Tk()
     root.resizable(False, False)
 
-    topbar = tkinter.Label(root)
-    topbar['bg'] = D_GREY
+    topbar = tkinter.Frame(root, bg=D_GREY, height=square_size*2, relief = 'ridge', borderwidth=1)
     topbar.grid(row=0, column=0, sticky='we')
+
+    topbar_next_block = tkinter.Label(topbar)
+    topbar_next_block.grid(column=2, columnspan=2, sticky='w')
```

Comment:
Because there's nothing else in the topbar, the column and columnspan don't matter, and defaults are fine:

```suggestion
    topbar_next_block.grid(sticky='w')
```

Reply:
Adjusted